Source/WebCore/dom/TreeScope.cpp

```diff
@@ -572,4 +577,165 @@ ExceptionOr<void> TreeScope::setAdoptedStyleSheets(Vector<RefPtr<CSSStyleSheet>>
     return ensureAdoptedStyleSheets().setSheets(WTFMove(sheets));
 }
 
+struct SVGResouresMap {
+    WTF_MAKE_NONCOPYABLE(SVGResouresMap); WTF_MAKE_FAST_ALLOCATED;
+public:
+    SVGResouresMap() = default;
+
+    MemoryCompactRobinHoodHashMap<AtomString, WeakHashSet<SVGElement, WeakPtrImplWithEventTargetData>> pendingResources;
+    MemoryCompactRobinHoodHashMap<AtomString, WeakHashSet<SVGElement, WeakPtrImplWithEventTargetData>> pendingResourcesForRemoval;
+    MemoryCompactRobinHoodHashMap<AtomString, RenderSVGResourceContainer*> resources;
+};
+
+SVGResouresMap& TreeScope::svgResouresMap() const
```

Comment:
Same "resoures" typo in this function name.

Reply:
Fixed.